enzyme/Enzyme/ActivityAnalysis.cpp

```diff
@@ -104,6 +116,20 @@ const std::set<std::string> InactiveGlobals = {
     "stderr",
     "stdout",
     "stdin",
+    "_ZSt3cin",
+    "_ZSt4cout",
```

Comment:
Add cerr?

Reply:
Seem to already work out of the box with these settings, but added a test.

Reply:
done